modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -138,11 +135,10 @@ The following section describes how to perform a logout.
 .Android
 ****
 
-. Get an instance of the auth service and retrieve the current user:
+. Retrieve the current user:
 +
 [source,java]
 ----
-AuthService authService = MobileCore.getInstance().getService(AuthService.class);
 UserPrincipal currentUser = authService.currentUser();
```

Comment:
Question. Do we need to provide info how authService if being created here. Old snippet done that.

Reply:
There is a section called `Initializing the SDK` with instruction how to ceate an `authService`